### PR TITLE
Use NVR as a suffix for the remote source archive

### PR DESCRIPTION
**Pull request by athos-ribeiro**
* Relates to OSBS-8137
* Relates to FACTORY-5450

Signed-off-by: Athos Ribeiro <athos@redhat.com>

Maintainers will complete the following section:
- [ ] Commit messages are descriptive enough
- [ ] "Signed-off-by:" line is present in each commit
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request includes link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/plugins/exit_koji_import.py [hunk 1]
```diff
@@ -501,8 +501,9 @@ def add_log_type(output):
             output_files.append(output_file)
 
         # add remote source tarball and remote-source.json files to output
+        nvr = '{}-{}-{}'.format(build['name'], build['version'], build['release'])
```
Comment (athos-ribeiro) [inline atomic_reactor/plugins/exit_koji_import.py]: Am I missing any cases where any of the NVR components would not be available at this point?

Comment (rcerven) [reply atomic_reactor/plugins/exit_koji_import.py]: get_build will always return nvr so should be fine

File: atomic_reactor/koji_util.py [hunk 2]
```diff
@@ -360,13 +360,14 @@ def get_image_output(workflow, image_id, arch):
...
 
-def get_source_tarball_output(workflow):
+def get_source_tarball_output(workflow, nvr):
     plugin_results = workflow.prebuild_results.get(PLUGIN_RESOLVE_REMOTE_SOURCE) or {}
     remote_source_path = plugin_results.get('remote_source_path')
     if not remote_source_path:
         return None
 
-    metadata = get_output_metadata(remote_source_path, REMOTE_SOURCES_FILENAME)
+    koji_archive_filename = '{}-{}'.format(nvr, REMOTE_SOURCES_FILENAME)
```
Comment (chmeliik) [inline atomic_reactor/koji_util.py]: The commit message says "Use NVR as a **suffix**", wouldn't this be a prefix?

**Comment by athos-ribeiro** [comment]
Dropping this since this is no longer needed

**Pull request closed without merging**